packages/xo-server/src/xo-mixins/authentication.js

```diff
@@ -193,6 +193,14 @@ export default class {
     }
   }
 
+  async deleteAllTokens(id, except) {
+    return Promise.all(
+      (await this.getAuthenticationTokensForUser(id)).map(
+        ({ id }) => id !== except && this.deleteAuthenticationToken(id)
```

Comment:
Unfortunately this does not work, it does not close existing sessions :slightly_frowning_face:

Reply:
But fixing this will be more complicated, maybe we should merge this as a start.